rllib/models/tests/test_base_model.py

```diff
@@ -0,0 +1,158 @@
+import unittest
+
+from ray.rllib.models.base_model import (
+    UnrollOutputType,
+    Model,
+    RecurrentModel,
+    ForwardOutputType,
+)
+import numpy as np
+from ray.rllib.models.temp_spec_classes import TensorDict, SpecDict
+
+
+class NpRecurrentModelImpl(RecurrentModel):
```

Comment:
can you explain in a comment what you are planning on doing with the RecurrentModel extension, so that it is easy for the next maintainer to understand what to expect?